NSSCodeForNashville/NashvilleHistory
# Ddw art markers

## Status
**READY/IN DEVELOPMENT**

## Documentation

[X ] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description

The goal of the code in this pull request is to add the art markers available via Nashville Gov's Art in Public Places and Metro Public Art APIs to the application. 

## Resolves Issue Number

#31

## Deploy Notes
Notes regarding deployment the contained body of work. 

```sh
git pull --prune
git checkout ddw-art-markers
ionic serve --lab
```

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1. Make sure that the art markers near you are showing up in the list of places near you
1. Make sure that something descriptive is showing up on the card (there are differences in how the data comes back from each API e.g. MarkerCard.description doesn't yield results for every entry). I tried to account for these differences but I was only able to test the markers near me.

## Impacted Files in Application
List general components of the application that this PR will affect:

* markers.html
* MarkersCtrl.js
* MarkerCardsFact.js
